bridgesample/src/main/java/com/livefront/bridgesample/scenario/activity/LargeDataActivity.kt

```diff
@@ -26,6 +28,12 @@ class LargeDataActivity : BridgeBaseActivity() {
             setHeaderText(R.string.large_data_header)
             generatedBitmap = savedBitmap
             onBitmapGeneratedListener = { savedBitmap = it }
+            if (getArguments(this@LargeDataActivity).infiniteBackstack) {
+                onNavigateButtonClickListener = {
+                    startActivity(getNavigationIntent(this@LargeDataActivity,
+                            getArguments(this@LargeDataActivity)))
+                }
```

Comment:
Formatting-wise let's do

```kotlin
startActivity(
        getNavigationIntent(this@LargeDataActivity,getArguments(this@LargeDataActivity))
)
```

or

```kotlin
startActivity(
        getNavigationIntent(
                 this@LargeDataActivity,
                 getArguments(this@LargeDataActivity)
         )
)
```

Same for the call below.

Reply:
I went with the second one, the first was still quite long.